supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -163,13 +177,79 @@ async def create_new_session(
         request.set_session(new_session)
         return new_session
 
+    async def assert_claims(
+        self,
+        session: SessionContainer,
+        claim_validators: List[SessionClaimValidator],
+        user_context: Dict[str, Any],
+    ) -> None:
+        log_debug_message(
+            "get_session: required validator ids %s",
+            ",".join([c.id for c in claim_validators]),
+        )
+        await resolve(session.assert_claims(claim_validators, user_context))
+        log_debug_message("get_session: claim assertion successful")
+
+    async def validate_claims_for_session_handle(
+        self,
+        session_info: SessionInformationResult,
+        claim_validators: List[SessionClaimValidator],
+        user_context: Dict[str, Any],
+    ) -> Union[ValidateClaimsOkResult, SessionDoesntExistError]:
+        original_session_claim_payload_json = json.dumps(
+            session_info.access_token_payload
+        )
+
+        new_access_token_payload = await update_claims_in_payload_if_needed(
+            claim_validators,
+            session_info.access_token_payload,
+            session_info.user_id,
+            user_context,
+        )
+
+        if json.dumps(new_access_token_payload) != original_session_claim_payload_json:
+            await self.merge_into_access_token_payload(
+                session_info.session_handle,
+                new_access_token_payload,
+                user_context,
+            )
```

Comment:
if this function returns false, we need to return SessionDoesntExistError

Reply:
Fixed.

I think this is missing from the Node PR as well.